source/assets/sass/components/_alerts.scss

```diff
@@ -4,15 +4,15 @@
 @use '../visual-design/typography';
 
 .sl-c-alert {
-  padding: {
-    top: functions.sl-px-to-rem(12px);
-    bottom: functions.sl-px-to-rem(12px);
-  }
-
-  text-align: center;
-  background: color.adjust(theme.$sl-color--hopbush, $lightness: -10%);
+  background: var(
+    --alert-bg,
```

Comment:
For component-specific custom properties, would it make sense to keep their definition in the component file?

Reply:
@mirisuzanne That alert-bg color does not exist other than in line 53, where it is first defined and used for both the info and special alerts. I don't know see any sl-c-alert--info in use as of now. I don't know if you are asking something else though?

Reply:
oh, I see. Nope, I was misreading this. Looks good!